### ISSUE TITLE
Debug controls does not appear if Go compilation fails while debugging

### ISSUE THREAD

**Issue by lirao** [Microsoft/vscode#5922]
- VSCode Version: 1.1.0-insider
- OS Version: Mac OSX 10.11.4

Steps to Reproduce:

1. Install Go
2. Install the vscode-go extension
3. Install the Go debugger
4. Create a new folder with hello.go that doesn't compile
``` 
package main
func main() {
    blah
}
```

5. Go to the debug tab and run debug.
6. The debug console will tell you there's a compile problem. There's a running horizontal blue line below the debug menu, which seem to show that the debug process is still going on. But since I can't stop the debugger, so the only thing I can do is close and reopen the project.
<img width="980" alt="screen shot 2016-04-27 at 4 56 19 pm" src="https://cloud.githubusercontent.com/assets/6883412/14871669/52114722-0c99-11e6-9d95-6362718294b0.png">

This bug only happened in the lastest Insiders build. Debugging worked fine in the previous version of insiders and in the stable 1.0.0 build, so I thought this bug is best posted here. I haven't tried vscode with anything other than go, so I'm not sure if it's just a language specific issue.

In the previous Insiders and stable versions, a compile failure will show a debug console, and the debugger controls at the top of the screen, with the option to stop the debugger.
<img width="870" alt="screen shot 2016-04-27 at 4 57 20 pm" src="https://cloud.githubusercontent.com/assets/6883412/14871719/e4596326-0c99-11e6-8cb2-528fc918ba4a.png">

**Comment by isidorn** [comment]
Sounds like a regression, assigning to april to investigate

**Comment by isidorn** [comment]
The issue was that we changed when the debug controls show, I changed that back that they are shown also if you are in the initalizing state. So in you scenario you will be able to stop debug more easily.

As for showing the debug console, we have tuned that this sprint, and we are only revealing the console on the first debug launch since users were complaing the console was too distracting and spammy.
